cryosparc/dataset.py

```diff
@@ -1552,6 +1653,18 @@ def _reset(self, data: Optional[Data] = None):
     def _ipython_key_completions_(self):
         return self.fields()
 
+    def create_summary(self, result_names_to_types_map):
```

Comment:
Why put this here and not in `compute/core/jobs` or `compute/jobs/run_context`?

Reply:
good question, I think it's that at the time I thought creating a summary from a dataset was more `Dataset` related rather than `job` related, but I didn't want to create a `core/dataset` module, so I did this instead.

I can move this to `core/jobs` if you think it's worth keeping `Dataset` shorter